### DISCUSSION TITLE
Please let us drag & drop iterations to change order

### DISCUSSION THREAD

**Discussion by daniandl** [community/community#58242] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

It is cumbersome needing to rename iterations and then manually re-assigning in task in the sprints.
Jira lets you just drag one iteration above the other. It's also not possible to change the date of a sprint to the past, so there is no way to re-order at the moment.

### Guidelines

- [x] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by MrChocolateMoose** [comment]
I agree! If you are using iterations for RELEASES then adding a release in the middle becomes very problematic.

**Comment by Vikfreud** [comment]
Fully agree! Super annoying, that that`s not possible. And should also be also very quickly realizable.

**Comment by IFW-OMS** [comment]
I agree that we need more options with iteration based planning, specifically being able to re-order or add an iteration between two existing ones is a must when using them for release planning.
at present this can only be achieved by moving tasks to new iterations to free up one 'in the middle' which is a huge amount of work and prone to errors.

**Comment by michalmikolajczyk** [comment]
I also need that feature to be able to reorder iterations

**Comment by BriBoRW** [comment]
Adding another vote for this one.

**Comment by AmmarHaddadi** [comment]
any update on this?
